client/stylesheets/_toCat.scss

```diff
@@ -5393,6 +5401,10 @@ On click the images will be displayed at normal size to complete the effect
     margin: 0 !important;
 }
 
+.optionsItems li {
```

Comment:
Merge rule `.optionsItems li` with rule on line 5400<br>Selector `optionsItems` should be written in lowercase with hyphens